src/pages/holofuel/PrimaryLayout/PrimaryLayout.js

```diff
@@ -0,0 +1,18 @@
+import React, { useContext } from 'react'
```

Comment:
considered calling these components `HFPrimaryLayout`, `HFDashboard`, `HFHeader` etc, so that it's easier to distinguish between them when doing fuzzy search, but in the end I decided not to. I think it will just cause more mess than not, especially when separating this out into its own repo.

Reply:
agree, good call.